ui/restoredlg.ui

```diff
@@ -225,11 +225,14 @@
     <item>
      <widget class="QTextEdit" name="confirm_text_edit">
       <property name="html">
-       <string>
-&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
+       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
```

Comment:
```suggestion
       <string>&lt;!DOCTYPE html&gt;
```

The other HTML DOCTYPEs are deprecated.

Reply:
this is generated by designer-qt, so you can report this upstream. Editing it manually is a bad idea. Same to your comments on https://github.com/QubesOS/qubes-desktop-linux-menu/pull/50

Reply:
I did not realize this was generated code.

Reply:
while in this file it might not be obvious, the ones in the other PR literally have "generated by XYZ software" in them...